.github/fabricbot.json

```diff
@@ -3006,6 +3006,130 @@
       ]
     }
   },
+  {
+    "taskSource": "fabricbot-config",
```

Comment:
```suggestion
```

This should be removed. We inject that from tasks generated from the dotnet/fabricbot-config tooling so that the scripts can detect which tasks are hand-authored and which are generated.

Also, if you could move this task to be immediately before the first task annotated with the `taskSource`, then regenerating from the scripts will retain the ordering and future merges will be cleaner. (That guidance wasn't written anywhere, sorry).